.gitignore

```diff
@@ -21,3 +21,6 @@ instance/
 .coverage
 htmlcov/
 *.spec
+league-rpc-env/
+
+getConnectDetails.py
```

Comment:
What is this file, and why do we need to ignore it?

Reply:
league-rpc-env is the default name for the python env according to your readme

getConnectDetails.py is a small script grabbing the LCU connect details from the process and prints them
This allowes me to easily copy them over to Insomnia for local testing of the endpoints
, not related to the Project, but only locally for my dev requirements. 
And its just a .gitignore anyway.  - If it matters, i'll remove it from the file.

Reply:
league-rpc-env makes sense to add here, sure. 

Not sure what getConnectDetails.py has inside of it.. maybe it's something useful during development and doesn't need to be ignored by git?

Reply:
```python
import psutil


def get_process_command(process_name: str) -> str:
    """
    Get process command of a process.
    """
    for proc in psutil.process_iter():
        try:
            if process_name.lower() in proc.name().lower():
                return proc.cmdline()
        except (psutil.NoSuchProcess, psutil.AccessDenied, psutil.ZombieProcess):
            pass
    return False


def get_lcu_connect_details(proc_name: str) -> bool:
    """
    Get LCU connect details from command.
    """
    cmd = get_process_command(proc_name)
    if not cmd:
        return False
    for arg in cmd:
        if "--remoting-auth-token=" in arg:
            print(arg.split("=")[1])
        elif "--app-port=" in arg:
            print(arg.split("=")[1])
    return True

get_lcu_connect_details(proc_name="CrBrowserMain")

```

Just a small simple prog to grab the cmdline from the process and print it.